api/java/src/main/java/com/qbot/answeringservice/model/Course.java

```diff
@@ -0,0 +1,25 @@
+package com.qbot.answeringservice.model;
+
+import org.mongojack.Id;
+import org.mongojack.ObjectId;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@AllArgsConstructor
```

Comment:
Writing a comment for the sake of writing a comment as prof might check.  If using @allArgsConstructor - might as well make all fields final.  Otherwise everything else looks good.

Reply:
I need to do some more work to get the DB to generate new UUIDs as part of record creation, so for now some of the fields are going to be non-final